site/content/en/docs/Reference/Transformers/AnnotationsTransformer.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "AnnotationsTransformer"
+linkTitle: "AnnotationsTransformer"
+date: 2023-07-21
+weight: 2
+description: >
+  Specification of AnnotationsTransformer behavior
+---
+<!-- [kustomization reference]: https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/ -->
+
+<!-- TODO(annasong): What is "weight"? What is this "kustomization reference"? -->
+<!-- TODO(annasong): Add Reference, Transformers, Utility/FieldSpec page descriptions -->
+
+## _AnnotationTransformer_
+
+### Usage via `kustomization.yaml`
+
+#### field name: `commonAnnotations`
+
+Adds annotations (non-identifying metadata) to
+all resources. Like labels, these are key value
+pairs.
```

Comment:
I was just wondering from a long time ago, Why do some people make a new line before a dot at the end of one sentence in not long English text?
I think the unnecessary new line in the documents and comments are hard to read (especially who is a non-native English speaker using translation tools).

Reply:
@koba1t you make a very good point that I never considered! 

This is a practice I blindly applied from coding. I try to limit my lines to [80 characters](https://softwareengineering.stackexchange.com/questions/148677/why-is-80-characters-the-standard-limit-for-code-width) for ease of display on a smaller terminal windows.

I have since modified this PR so that a single sentence stays on the same line. Does this help?

Reply:
Thanks for asking about my idiot question!
I understand why people use short lines to follow their coding style. (And I think I have heard of the rule of limits to 80 characters in one line on the Linux project.)

> I have since modified this PR so that a single sentence stays on the same line. Does this help?

Thanks for your fix!
I think this style is easy to read!